<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAddItem</class>
 <widget class="QDialog" name="QAddItem">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QDisplayLabel" name="lb_liveStream">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>490</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Live Stream</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_getFrame">
       <property name="text">
        <string>Start capturing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_calibrate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_editItem">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit Item</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDisplayLabel</class>
   <extends>QLabel</extends>
   <header location="global">qdisplaylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
